Common/src/CConfig.cpp

```diff
@@ -5197,6 +5194,20 @@ void CConfig::SetPostprocessing(SU2_COMPONENT val_software, unsigned short val_i
       nTurbVar = 2; break;
   }
 
+  /*--- Postprocess SST_OPTIONS into structure. ---*/
+  if (Kind_Turb_Model==TURB_MODEL::SST || Kind_Turb_Model==TURB_MODEL::SST_SUST){
+    sstParsedOptions = ParseSSTOptions(SST_Options, nSST_Options);
```

Comment:
So, here I would copy SST_Options into a std vector and push_back SUST (if SST_SUST) and UQ (if using_uq), then you pass the vector for parsing, duplicated don't matter.

Reply:
Do you mean create a vector<SST_OPTIONS> and pass that as argument to ParseSSTOptions (instead of SST_OPTIONS itself)?